pkgs/development/tools/kustomize/default.nix

```
@@ -2,7 +2,7 @@
 
 buildGoModule rec {
   pname = "kustomize";
-  version = "4.4.0";
+  version = "4.4.1";
   # rev is the commit of the tag, mainly for kustomize version command output
   rev = "9e8e7a7fe99ec9fbf801463e8607928322fc5245";
```

https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.4.1

```suggestion
  rev = "b2d65ddc98e09187a8e38adc27c30bab078c1dbf";
```